week3/final/robot-text/robot.js

```diff
@@ -1,21 +1,22 @@
-(function () {
+((function () {
```

Comment:
You don't need these extra parentheses. As I explained earlier this is sufficient:

```js
(function () {
    // all your other code
})();

```